api/v1alpha1/addresspool_types.go

```diff
@@ -42,10 +42,19 @@ type AddressPoolSpec struct {
 	AutoAssign bool `json:"autoAssign,omitempty"`
 }
 
+// AddressPool Services information
+type AddressPoolSvcStats struct {
+	// Service Name
+	Service string `json:"service,omitempty"`
+
+	// Service's Allocated Externat IP address
+	ExternalIP string `json:"external-ip,omitempty"`
```

Comment:
I think it should be `externalIP` link the service object

	// externalIPs is a list of IP addresses for which nodes in the cluster
	// will also accept traffic for this service.  These IPs are not managed by
	// Kubernetes.  The user is responsible for ensuring that traffic arrives
	// at a node with this IP.  A common example is external load-balancers
	// that are not part of the Kubernetes system.
	// +optional
	ExternalIPs []string `json:"externalIPs,omitempty" protobuf:"bytes,5,rep,name=externalIPs"`